### PR TITLE
Ankle socks

### PR DESCRIPTION
**Pull request by FuelType-Memes**
#### Summary
```SUMMARY: Content "Adds new item 'Ankle socks'"```

#### Purpose of change
Ankle socks are popular type of socks, used for sports and with some footwear. I also wanted an alternative to socks that I can wear in the summer, so those provide no warmth bonus

#### Describe the solution
Added a new item, a recipe for said item. Modified loot tables and also some starting "sports" professions

#### Describe alternatives you've considered
Going sockless in summer and **RUINING** immersion

#### Testing
Game compiles. Spawned in new socks, crafted them. Spawned as a bionic runner to test professions

#### Additional context
My first item addition!

### CODE REVIEW

File: data/json/recipes/armor/feet.json [hunk 1]
```diff
@@ -422,6 +422,17 @@
...
+  {
+    "result": "socks_ankle",
+    "type": "recipe",
+    "category": "CC_ARMOR",
+    "subcategory": "CSC_ARMOR_FEET",
+    "skill_used": "tailor",
+    "time": "8 m",
+    "autolearn": true,
+    "using": [ [ "sewing_standard", 4 ] ],
+    "components": [ [ [ "rag", 1 ] ] ]
```
Comment (Night-Pryanik) [inline data/json/recipes/armor/feet.json]: 8 minutes feels like way too fast for a pair of socks. Also, 1 rag is too low. There should be at least 2 rags.

Comment (FuelType-Memes) [reply data/json/recipes/armor/feet.json]: Basic socks take 10m to craft, these are smaller so I made them take a little bit less time. Basic socks also take 2 rags, but I agree, ankle socks should take 2 rags too. Maybe less thread?

Comment (LaVeyanFiend) [reply data/json/recipes/armor/feet.json]: Seems @Rail-Runner should release the next part of the tailoring re-work then to fix the sock making time issue.

**Comment by I-am-Erk** [comment]
Needs linting

**Pull request merged**